script/States/App/LTheory.lua

```diff
@@ -111,13 +111,13 @@ function LTheory:onUpdate(dt)
     self.player:getRoot():update(dt)
     self.canvas:update(dt)
 
-    HmGui.Begin(self.resX, self.resY, InputInstance) -- required for HmGui.Draw() to work without crashing
-    HmGui.End(InputInstance)
+    Gui:beginGui(self.resX, self.resY, InputInstance) -- required for Gui:draw() to work without crashing
```

Comment:
This still feels a bit weird that we have to do this. Is there no way to circumvent setting a begin and end like that?

Reply:
I'll check this when I start testing Lua part.